MovieSharing/VideoManager.swift

```diff
@@ -9,20 +9,30 @@ import Foundation
 
 class VideoManager{
     
+    // MARK: - Types
+    //FIX ME:
```

Comment:
- [ ] since you already have ticket for more proper error handling... it is in general good practice to put a link into a comment with a TODO or FIX ME as well, so link that ticket in the respective comment please